src/scale.ts

```diff
@@ -382,9 +382,13 @@ export interface Scale {
    */
   domain?: Domain;
 
+
+  // Hide because we might not really need this.
```

Comment:
As said on slack, it's too much work to remove it.  Plus, we may need it in the future. 
Totally not worth the time now.